test/e2e/custom_wait_rules_test.go

```diff
@@ -222,34 +260,37 @@ metadata:
 		}
 		patch := `[{ "op": "replace", "path": "/status/currentState", "value": "Running"}]`
 		PatchClusterResource("CronTab", "my-new-cron-object-1", env.Namespace, patch, kubectl)
-	}()
+	}
 
 	logger.Section("deploy resource with current state as progressing", func() {
 		kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", crdApp}, RunOpts{StdinReader: strings.NewReader(crd)})
+		go patchConfigMap()
 
-		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name}, RunOpts{
-			StdinReader: strings.NewReader(yaml)})
+		kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name}, RunOpts{StdinReader: strings.NewReader(cr)})
+
+		//remove age column from output, as we can't rely on it
+		columns := "namespace,name,kind,op,'op strategy','wait to','reconcile state','reconcile info'"
+		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name, "--column", columns}, RunOpts{
+			StdinReader: strings.NewReader(yaml + cr)})
```

Comment:
We don't need to deploy twice now as the default value for status would be populated and we don't need the first patch.
Also, we are not asserting on the `Age` column and it's just empty so I am slightly confused with the comment `remove age column from output, as we can't rely on it`
```suggestion
		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name}, RunOpts{StdinReader: strings.NewReader(yaml + cr)})
```

Reply:
In the actual output we would have the age, which can differ from time to time, which would lead to a flaky test